### PR TITLE
Fixing some CodeQL errors

### PR DESCRIPTION
**Pull request by zuphilip**
Needs some testing and probably I will expand this PR further...

### CODE REVIEW

File: isbn/rendering.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ function renderRVK(rvkArray) {
     } else {
         for (var i=0; i<rvkArray.length; i++) {
             var rvk = rvkArray[i];
-            var rvkUrl = 'https://rvk.uni-regensburg.de/notation/' + rvk.replace(' ', '%20');
+            var rvkUrl = 'https://rvk.uni-regensburg.de/notation/' + encodeURI(rvk.replace);
```
Comment (stweil) [inline isbn/rendering.js:35]: Should that be `encodeURI(rvk)`?

Comment (zuphilip) [reply isbn/rendering.js:35]: Uii, yes, you are right...

File: isbn/suche.html [hunk 2]
```diff
@@ -273,14 +274,18 @@
...
-        }
-    });
+    var alle = ["swb", "k10plus", "hebis", "b3kat", "hbz", "swiss", "obvsg"];
+    if (alle.includes(verbund)) {
+        $.getJSON(verbund+".php?ppn="+ppn+"&format=json", function(data){
+            if (data['isbn'].length > 0) {
+                var isbnString = data['isbn'].join(",");
+                isbnEingabe(isbnString);
+            } else {
+                var suchLink = $('<a/>').attr('href', verbund + '.php?ppn=' + ppn).text('Daten zum Titel');
```
[the remark this answers is not in the document]
Comment (zuphilip) [reply isbn/suche.html:284]: Hm... I don't know how to fix this last issue :-(

I already restricted the values for `verbund` in my previous commit, which is also used here and probably causes this issue. But is this a real problem and how to fix it?

**Comment by zuphilip** [comment]
Testing looks and therefore I am merging now.

**Pull request merged**